doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -92,10 +92,10 @@ Steps for Using VxWorks as User VM
 
    You now have a virtual disk image with bootable VxWorks in ``VxWorks.img``.
 
-#. Follow XXX to boot the ACRN Service VM.
+#. Follow `Build and Install ACRN on Ubuntu
+   <https://projectacrn.github.io/latest/getting-started/rt_industry_ubuntu.html#build-and-install-acrn-on-ubuntu>`_
```

Comment:
Same as above, please link to the `.rst` file.

Reply:
Done!